test/sqllogictest/chbench.slt

```diff
@@ -430,7 +430,8 @@ ORDER BY o_ol_cnt
 %4 =
 | Join %0 %3 (= #0 #8) (= #1 #9) (= #2 #10) (= #4 #11)
 | | implementation = Differential %0 %3.(#0, #1, #2, #3)
-| | demand = (#6)
+| | demand = (#4, #6)
```

Comment:
It seems the predicate was lifted all the way up and then it could not be returned to its previous place.